### ISSUE TITLE
Deposit repository artifacts on Zenodo Sandbox

### ISSUE THREAD

**Issue by sdruskat** [hermes-hmc/workflow#130]
- `hermes` uploads actual files to Zenodo Sandbox
- Users need to be able to define what's being uploaded

**Comment by poikilotherm** [comment]
Probably we should offer two ways to hand over the files  that a user wants to deposit: either as arguments to `hermes deposit` and/or via config file.

Command line usage is eased by enabling the arguments variant.

**Comment by zyzzyxdonta** [comment]
The files API can be used to delete existing files. GET `deposit["links"]["files"]` for a list of `files` (names, checksums, ...), then DELETE `file["links"]["download"]` for each `file` in the list.

Upload new files using the bucket API:

```python
for file in files:
    with open(file, "rb") as file_data:
        response = session.put(f"{bucket_url}/{file}", data=file_data)
        response.raise_for_status()
```

Consider whether the file name needs to be URL-encoded for this to work.

**Comment by poikilotherm** [comment]
I did a bit of a [hacky approach to get this done for the demo at deRSE23](https://github.com/hermes-hmc/workflow/tree/deRSE23), but we should do this cleaner and nicer in a proper pull request.

**Comment by sdruskat** [comment]
CLosed via #185.
